### DISCUSSION TITLE
Legacy Song Progress

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#19394] [Ideas] [answered]
Some feedback concerns the current song progress which takes up too much space even with the difficulty graph contracted and they prefer stable's song progress. I have a working demo which I plan on implementing but for now I'll leave it as a proposal.

I intend to replace the song progress and make use of the pie chart styled progress for legacy skins. With its minimal appearance, controls for seeking are not available for this style. Although it is still possible through key presses.

https://user-images.githubusercontent.com/20495991/180944452-e115cc7a-8bc3-48fd-a101-c343a763aa37.mp4

![image](https://user-images.githubusercontent.com/20495991/180944560-90420101-ec5a-435d-8381-7a08b70fc9d7.png)

Everything is eyeballed to what I think is close enough. Will proceed to a PR if this is granted approval.

**Comment by peppy** [comment, marked as answer]
There's already tens of issues covering this being missing (most recent being https://github.com/ppy/osu/discussions/17496).

**Reply by smoogipoo** [reply]
To clarify, the poster has already implemented it within lazer and is asking for approval to PR.

Which is fine. I think there's an epic issue for this somewhere also.

**Reply by frenzibyte** [reply]
https://github.com/ppy/osu/issues/18893 is the only one I can think of.
